.changeset/sixty-deers-suffer.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This is an example of an empty changeset - one that does not cause a version change, but is still documented in the changelog. See https://github.com/changesets/changesets/issues/145 for more details